src/plugins/new-ui/public/webcomponents/depth.html

```diff
@@ -16,9 +16,9 @@
         margin-top: -0.6vw;
       }
       #depth {
-        font-size: 1.8vw;
+        font-size: 1.35vw;
         font-weight: bold;
-        text-align: center;
+        text-align: left;
```

Comment:
As an alternative, you could put the fresh/salt in the lower left and the units in the lower right...